### PR TITLE
add encryption to docs

### PR DESCRIPTION
**Pull request by khaya-zulu**

### CODE REVIEW

File: docs/01-overview.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ Seed scripts are tedious to write and maintain, inaccurate and a tiny representa
...
 
-Snaplet is a self-service workflow that allows developers to copy whilst transforming a data, which can then be shared and restored into any development environment.
+Snaplet is a self-service workflow that allows developers to copy whilst transforming their data, which can then be shared and restored into any development environment.
 
 ### Examples
 
 1. Join a new team, run `snaplet snapshot restore` and get a copy from the evening before
 2. See a bug in production? run `snaplet snapshot restore --new` and restore an "on demand" snapshot in an instant
-3. Write E2E tests again actual data
+3. Write E2E tests aginst actual data
```
Comment (peterp) [inline docs/01-overview.md]:
```suggestion
3. Write E2E tests against actual data
```

File: docs/04-references/snapshots.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ Will restore the latest snapshot. This will download the snapshot, create the sc
 
 That's it! You've just created and restored your first snapshot.
 
+## Restoring encrypted snapshots
+
+If you have encryption enabled in Snaplet Cloud and want to restore, you will need to provide us with a private key (generated with your public key) which you store in your `.snaplet/id_rsa` directory file. Your private key will be used to decrypt your table data.
```
Comment (peterp) [inline docs/04-references/snapshots.md]: We'll need to rework this sentence.

```suggestion
In order to decrypt snapshots (that were encrypted) you'll need to provide the private key. Store this file in `.snaplet/id_rsa` or use the `SNAPLET_PRIVATE_KEY` environmental variable.
```

Comment (peterp) [reply docs/04-references/snapshots.md]: @khaya-zulu Does `SNAPLET_PRIVATE_KEY` work?

**Review by peterp — APPROVED** [review]
I made a suggestion. Is there an extension or something installed that modifies other parts of the files when you're editing them?

File: docs/04-references/snapshots.md (hunk 2 above)
Comment (khaya-zulu) [reply docs/04-references/snapshots.md]: No it isn't included

File: docs/01-overview.md [hunk 3]
```diff
@@ -4,33 +4,44 @@ slug: /
...
+Snaplet is a self-service workflow that allows developers to copy whilst transforming their data, which can then be shared and restored into any development environment.
+
+### Examples
+
+1. Join a new team, run `snaplet snapshot restore` and get a copy from the evening before
 2. See a bug in production? run `snaplet snapshot restore --new` and restore an "on demand" snapshot in an instant
-3. Write E2E tests again actual data
+3. Write E2E tests against actual data
 4. Spin up pull request/ deploy preview databases with accurate data
+>>>>>>> 53a809dcd79cfefa037dafaf002e14d5e1383406
```
Comment (peterp) [inline docs/01-overview.md]: Oh no.

**Review by justinvdm — APPROVED** [review]

**Pull request merged**